### PR TITLE
Automate docker image build

### PR DESCRIPTION
**Pull request by saratry**
With this module you can activate the profile docker-hub-images in order
to build the images of AxonServer SE. This doesn't perform the push (yet).

### CODE REVIEW

File: docker/axonserver-docker-hub/src/main/resources/common/APACHE LICENSE v2.0.txt [hunk 1]
```diff
@@ -0,0 +1,202 @@
+
```
Comment (smcvb) [inline docker/axonserver-docker-hub/src/main/resources/common/APACHE LICENSE v2.0.txt]: I think we were supposed to only have the AxonIQ license in here?

Comment (bert-laverman) [reply docker/axonserver-docker-hub/src/main/resources/common/APACHE LICENSE v2.0.txt]: If we have any file which is under the Apache license in there, we need to add it.

Comment (smcvb) [reply docker/axonserver-docker-hub/src/main/resources/common/APACHE LICENSE v2.0.txt]: Fair, that makes sense.
Do you also know whether there is a file under the Apache license in axon-server-se?

Comment (saratry) [reply docker/axonserver-docker-hub/src/main/resources/common/APACHE LICENSE v2.0.txt]: @bert-laverman, @MGathier do you know if we have any files under the Apache license? if not I can remove it...

**Comment by saratry** [comment]
> AxonIQ license file is already available in root of Axon Server SE (LICENSE file).

Fixed. Now we have a single LICENSE file in root (that is copied in the docker images)

File: docker/axonserver-docker-hub/src/main/resources/docker-jdk11/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+FROM openjdk:11-jdk
+
+ENV AXONSERVER_HOME /opt/axonserver
+RUN mkdir -p ${AXONSERVER_HOME}/data
+
+COPY axonserver.jar ${AXONSERVER_HOME}/axonserver.jar
+COPY axonserver.properties ${AXONSERVER_HOME}/axonserver.properties
+COPY axonserver-cli.jar ${AXONSERVER_HOME}/axonserver-cli.jar
+COPY startup.sh ${AXONSERVER_HOME}/startup.sh
+COPY ["AXONIQ OPEN SOURCE LICENSE.txt", "${AXONSERVER_HOME}/AXONIQ OPEN SOURCE LICENSE.txt"]
```
Comment (MGathier) [inline docker/axonserver-docker-hub/src/main/resources/docker-jdk11/Dockerfile]: Name of license file changed as copied from another file.

File: docker/axonserver-docker-hub/src/main/resources/docker-jdk8/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+FROM openjdk:8-jdk
+
+ENV AXONSERVER_HOME /opt/axonserver
+RUN mkdir -p ${AXONSERVER_HOME}/data
+
+COPY axonserver.jar ${AXONSERVER_HOME}/axonserver.jar
+COPY axonserver.properties ${AXONSERVER_HOME}/axonserver.properties
+COPY axonserver-cli.jar ${AXONSERVER_HOME}/axonserver-cli.jar
+COPY startup.sh ${AXONSERVER_HOME}/startup.sh
+COPY ["AXONIQ OPEN SOURCE LICENSE.txt", "${AXONSERVER_HOME}/AXONIQ OPEN SOURCE LICENSE.txt"]
```
Comment (MGathier) [inline docker/axonserver-docker-hub/src/main/resources/docker-jdk8/Dockerfile]: Note that license file is changed.

File: docker/axonserver-docker-hub/src/main/resources/common/APACHE LICENSE v2.0.txt (hunk 1 above)
Comment (bert-laverman) [reply docker/axonserver-docker-hub/src/main/resources/common/APACHE LICENSE v2.0.txt]: As mentioned on Slack, the Roboto slab fonts are under the Apache 2.0 license

**Pull request merged**